language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Function.hs

```diff
@@ -264,16 +262,16 @@ getBuiltinFixN n = do
 
 -- | Get the fixed-point of a list of mutually recursive functions.
 --
--- > getBuiltinMutualFixOf _ [ FunctionDef _ fN1 (FunctionType _ a1 b1) f1
+-- > MutualFixOf _ [ FunctionDef _ fN1 (FunctionType _ a1 b1) f1
 -- >                         , ...
 -- >                         , FunctionDef _ fNn (FunctionType _ an bn) fn
 -- >                         ] =
 -- >     Tuple [(a1 -> b1) ... (an -> bn)] $
 -- >         fixN {a1} {b1} ... {an} {bn}
 -- >             /\(q :: *) -> \(choose : (a1 -> b1) -> ... -> (an -> bn) -> q) ->
 -- >                 \(fN1 : a1 -> b1) ... (fNn : an -> bn) -> choose f1 ... fn
-getBuiltinMutualFixOf :: ann -> [FunctionDef TyName Name ann] -> Quote (Tuple ann)
-getBuiltinMutualFixOf ann funs = do
+getMutualFixOf :: ann -> [FunctionDef TyName Name ann] -> Quote (Tuple ann)
```

Comment:
It is never allowed to escape `Quote` in a function that receives one or multiple PLC entities as arguments, because those can be open and we can return something open as well.

Reply:
It is also never allowed to escape `Quote` in a function that returns type or term variables as a part of its result, because those are just open things.